src/nvim/api/private/defs.h

```diff
@@ -107,7 +107,7 @@ typedef enum {
   kObjectTypeLuaRef,
   // EXT types, cannot be split or reordered, see #EXT_OBJECT_TYPE_SHIFT
   kObjectTypeBuffer,
-  kObjectTypeWindow,
+  kObjectTypeNvimWindow,
```

Comment:
could be just `kNvimWindow`, I don't think there's any ambiguity there.

Reply:
^ suggestion still stands. Full list of suggested names:

```
kNvimWin
kNvimTab
kNvimBuf
```

and for the types:

```
NvimWin
NvimTab
NvimBuf
```